laws.md

```diff
@@ -84,5 +82,10 @@ New Judges are not bound by the decisions of old Judges. New Judges may, however
 
 This rule takes precedence over every other rule determining the winner.
 
-**301.** Ammendment of rule **201.**: Each player may submit one proposal every twenty-four (24) hours, in any player order. Each proposal will remain for the 24 hour period to be voted on and discussed. At the end of the 24 hour period, the rule is either adopted, or rejected.
+**301.** Each player may submit one proposal every twenty-four (24) hours, in any player order. Each proposal will remain for the 24 hour period to be voted on and discussed. At the end of the 24 hour period, the rule is either adopted, or rejected.
 
+**303.** At the end of a proposal's 24 hour period, if the number of yay's is greater than the nay's the proposal is considered adopted. 
```

Comment:
You might want to add a clause in here that a vote is either a yay or nay. Then you get all the benefits of other laws that deal with votes. Right now how it is written people could leave multiple yay's or nay's.